FireRender.Maya.Src/Context/FireRenderContext.cpp

```diff
@@ -2601,8 +2588,7 @@ bool FireRenderContext::keepRenderRunning()
 	}
 
 	// check time limit completion criteria
-	long numberOfClicks = clock() - m_renderStartTime;
-	double secondsSpentRendering = numberOfClicks / (double)CLOCKS_PER_SEC;
+	double secondsSpentRendering = TimeDiffChrono(GetCurrentChronoTime(), m_renderStartTime) / 1000;
```

Comment:
is division by 1000 necessary? See comments below

Reply:
done